### PR TITLE
Add version command

### PR DESCRIPTION
**Pull request by Mackiovello**

### CODE REVIEW

**Comment by Mackiovello** [comment]
Build: https://teamcity.starcounter.org/viewQueued.html?itemId=72846

**Comment by Mackiovello** [comment]
> We don't want any kind of test?

We do. I'm not sure how though. Do you have an idea of how to test this?

**Comment by per-samuelsson** [comment]
> Do you have an idea of how to test this?

Integration test that grep output?

**Comment by Mackiovello** [comment]
> Integration test that grep output?

I'll refactor a bit first to make unit tests possible. And if that's enough, I can add integration tests as well. I'll ping you when I've finished the tests.

**Comment by Mackiovello** [comment]
@per-samuelsson I've added some tests now. Thank you for reminding me to add them

File: src/Starcounter.Nova.Cli/CommandLineApplicationBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+                new CreateCommand(_services),
+                new DeleteCommand(_services),
+                new ConfigCommand(_services),
+                new ListCommand(_services),
+                new HostCommand(_services),
+                new VersionCommand(_services)
+            };
+        }
+
+        public CommandLineApplicationBuilder AddCommand(ICommand command)
```
Comment (per-samuelsson) [inline src/Starcounter.Nova.Cli/CommandLineApplicationBuilder.cs]: Feel a bit bloated with AddCommand/AddCommands for such a trivial thing: adding items to a list ensuring they are unique. WDYT about merging them and/or using different backing, like `HashSet<T>` or generic dictionary (with name as key)?

Comment (per-samuelsson) [reply src/Starcounter.Nova.Cli/CommandLineApplicationBuilder.cs]: And regarding merging them, maybe `AddCommands(params ICommand[] commands)`?

**Comment by Mackiovello** [comment]
Running when the build passes: https://teamcity.starcounter.org/viewLog.html?buildId=73152&tab=buildResultsDiv&buildTypeId=Bifrost_UmbrellaIntegrationNovaBuild

**Pull request merged**